server/src/main/java/org/elasticsearch/common/util/SetBackedScalingCuckooFilter.java

```diff
@@ -39,13 +39,13 @@
  * An approximate set membership datastructure that scales as more unique values are inserted.
  * Can definitively say if a member does not exist (no false negatives), but may say an item exists
  * when it does not (has false positives).  Similar in usage to a Bloom Filter.
- *
+ * <p>
```

Comment:
I hate javadocs so much :(  optimizing rendered readability while sacrificing IDE readability :(

Thanks for fixing this :)